src/app/results/results.component.html

```diff
@@ -169,13 +169,16 @@
       </div>
     </div>
     <div class="feed container">
-      <div *ngFor="let item of items$|async" class="result">
+      <div *ngFor="let item of newsResponse" class="result">
         <div class="title">
-          <a class="title-pointer" href="{{item.link}}">{{item.title}}</a>
+          <a class="title-pointer" href="{{item.link}}" [style.color]="themeService.titleColor">{{item.title}}</a>
         </div>
         <div class="link">
-          <p>{{item.link}}</p>
+          <p  [style.color]="themeService.linkColor">{{item.link}}</p>
         </div>
+        <div class="description">
+          <p [style.color]="themeService.descriptionColor">{{item.pubDate|date:'MMMM d, yyyy'}} - {{item.description}}</p>
```

Comment:
keep spacing between pipe variables.

Reply:
This style is followed throughout the project.

Reply:
but it should be followed.